### DISCUSSION TITLE
Proxy 443 and 80 Port without certificate generation

### DISCUSSION THREAD

**Discussion by docmg** [NginxProxyManager/nginx-proxy-manager#2819] [Q&A]
Is it possible just to forward 443 and 80 Ports?
I have an container with symfony application served by caddy (template: https://github.com/dunglas/symfony-docker).
Caddy is currently managing the SSL certificates and the http to https redirects. Is it possible to forward both of this ports to the container and disable SSL-Certificate generation in npm?

**Comment by johnathanhawn** [comment]
If this is solved ill paypal $5
